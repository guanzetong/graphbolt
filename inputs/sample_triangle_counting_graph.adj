AdjacencyGraph
16
32
0
6
12
16
22
25
26
28
29
30
30
31
31
31
31
32
1
4
8
10
11
15
2
3
4
8
9
13
5
6
9
14
5
7
8
11
13
15
6
7
9
7
10
11
12
9
14
15
