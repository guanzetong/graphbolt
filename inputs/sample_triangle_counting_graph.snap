0   1
0   4
0   8
0   10
0   11
0   15
1   2
1   3
1   4
1   8
1   9
1   13
2   5
2   6
2   9
2   14
3   5
3   7
3   8
3   11
3   13
3   15
4   6
4   7
4   9
5   7
6   10
6   11
7   12
8   9
10  14
14  15